recipes/petsc4py/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set build = 0 %}
+{% set version = '3.7.0' %}
+{% set md5 = '816a20040a6a477bd637f397c9fb5b6d' %}
+{% set mpi = os.environ.get('MPI') or 'mpich' %}
+
+package:
+  name: petsc4py
+  version: {{version}}
+
+source:
+  fn: petsc4py-{{version}}.tar.gz
+  url: https://bitbucket.org/petsc/petsc4py/downloads/petsc4py-{{version}}.tar.gz
+  md5: {{md5}}
+
+build:
+  number: {{build}}
+  script:
+    - '$PYTHON setup.py build --petsc-dir=$PREFIX install'
+  skip: true  # [win]
+
+requirements:
+  build:
```

Comment:
Done. For reference for my other recipes, what does toolchain do and under what circumstances should I use it? All recipes that compile anything?